core/client/app/routes/posts.js

```diff
@@ -19,7 +18,8 @@ export default AuthenticatedRoute.extend(ShortcutsRoute, PaginationRouteMixin, {
 
         return this.get('session.user').then(function (user) {
             if (user.get('isAuthor')) {
-                paginationSettings.filter += `+author:${user.get('slug')}`;
+                paginationSettings.filter = paginationSettings.filter ?
+                    paginationSettings.filter + `+author:${user.get('slug')}` : `author:${user.get('slug')}`;
```

Comment:
Minor thing but this will likely throw a linter error once ember-suave is in place as it's using concatenation instead of interpolation. `${paginationSettings.filter}+author:${user.get('slug')}` would pass ok.